### PR TITLE
Add the 'less-is-more' workshopper

### PR DESCRIPTION
**Pull request by gnerkus**
I've built a workshopper about LESS called [`less-is-more`](https://github.com/gnerkus/less-is-more)

screen shot 2016-07-12 at 12 06 56 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -423,11 +423,16 @@ <h3 data-i18n="index-workshoppers-elective-header">Electives</h3>
...
         </div>
-         <div id="seneca-in-practice" class="workshopper">
+        <div id="seneca-in-practice" class="workshopper">
           <h4><a class="js-workshop-link" href="https://github.com/senecajs/seneca-in-practice" target="_blank">Seneca in practice</a></h4>
           <p data-i18n="workshopper-seneca-in-practice">Learn how to leverage SenecaJs microservice toolkit.</p>
           <code>npm install -g seneca-in-practice</code>
         </div>
+        <div id="less-is-more" class="workshopper">
+          <h4><a class="js-workshop-link" href="https://github.com/gnerkus/less-is-more" target="_blank">LESS is more</a></h4>
+          <p data-i18n="workshopper-node-debug-school">Learn the fundamentals of the LESS CSS preprocessor.</p>
```
Comment (riyadhalnur) [inline index.html]: value needs to be `workshopper-less-is-more` for the `data-i18n` attribute

**Comment by gnerkus** [comment]
I've fixed the `data-i18n` attribute.

**Comment by riyadhalnur** [comment]
Thank you! Merged 🚀

**Comment by gnerkus** [comment]
Thank you.

**Pull request merged**